### DISCUSSION TITLE
Execute binary compiled with AMD cpu on INTEL cpu

### DISCUSSION THREAD

**Discussion by LCGF00** [azerothcore/azerothcore-wotlk#5589] [General]
Hi

I wonder if there is any way to compile the emulator binary on an AMD hardware and then transfer the binary to a machine with an intel CPU, currently when I do this the "intel" server does not run the binary.

I'm using Ubuntu 20.04 on my "AMD" computer running on Windows 10 "WSL" and trying to send / run it on an intel cloud machine.

Thank you so much!

ps: i think its not the right place to ask for help,  i have created this post stackoverflow https://stackoverflow.com/questions/67353159/execute-binary-compiled-with-amd-cpu-on-intel-cpu

if this is not the appropriate place for the question please someone delete this post because I couldn't find an option to delete it

**Comment by Yehonal** [comment]
The only way is by using docker I'd say

**Reply by LCGF00** [reply]
hi
i figured whats happening...
the problem is just the patch who differ from compiled and target machine

my PC is like //home/luis/azerothcore and on targeted server is //home/admin/azerothcore

i changed my installation to math the server and now my binaries can run fine...
Windows 10 wsl can run azeroth on ubuntu 100000x better from docker, but needs manual "classic" installation, the compilation process take 80% fast instead of docker =P

thanks forthe response =)

**Reply by LCGF00** [reply]
I got stuck because when sending the file to the server in the cloud and trying to run it, no error or message was shown, there was simply a line break and nothing happened, some message alerting the wrong path would be good ...

**Reply by Yehonal** [reply]
Glad you solved it :) yeah we should provide a message indeed. 

However, If you use docker on wsl the compilation is fast as wsl without docker. 
Also consider to use the acore dashboard to install/compile. It's easier if you want to stay on wsl

**Reply by FrancescoBorzi** [reply]
hello, please post your solution to StackOverflow as an answer to your own question ;)

for general guidelines about how to ask for support, please read: https://www.azerothcore.org/wiki/How-to-ask-for-help

**Reply by LCGF00** [reply]
> hello, please post your solution to StackOverflow as an answer to your own question ;)
> 
> for general guidelines about how to ask for support, please read: https://www.azerothcore.org/wiki/How-to-ask-for-help

Done 👍
